src/components/videoCard/videoCard.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./videocard.css";
+import { Menuicon } from "../../assets/svg/menu";
+import { Unlike } from "../../assets/svg/unlike";
+function Videocard() {
+
+  
+  return (
+    <div className="card-container">
+      <img className="card-img" src="Images/videoImages/backpack.webp" />
+      <section className="card-description">
+        <section className="card-description-one">
+          <img className="card-logo-img" src="Images/hikeslogo.webp" />
+          <span className="card-title">
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Enim,
```

Comment:
I think you forgot to change this Lorem Ipsum.

Reply:
its itentional.